Neos.Utility.Files/Classes/Files.php

```diff
@@ -432,16 +431,27 @@ public static function unlink($pathAndFilename)
      * @param string $thousandsSeparator thousands separator of the resulting string
      * @return string the size string, e.g. "1,024 MB"
      */
-    public static function bytesToSizeString($bytes, $decimals = 0, $decimalSeparator = '.', $thousandsSeparator = ',')
+    public static function bytesToSizeString($bytes, int $decimals = null, string $decimalSeparator = null, string $thousandsSeparator = null): string
     {
-        if (!is_integer($bytes) && !is_float($bytes)) {
+        if (!is_int($bytes) && !is_float($bytes)) {
             if (is_numeric($bytes)) {
                 $bytes = (float)$bytes;
             } else {
                 $bytes = 0;
             }
         }
 
+        // FIXME: The defaults should again be in the function declaration and nulls not allowed. This is a breaking change though.
+        if ($decimals === null) {
+            $decimals = 0;
+        }
+        if ($decimalSeparator === null) {
+            $decimalSeparator = '.';
+        }
+        if ($thousandsSeparator === null) {
+            $thousandsSeparator = ',';
+        }
+
```

Comment:
Sure. Because we also want/need to accept nulls to not be breaking (see FilesTest::bytesToSizeStringDataProvider()), which we can not typehint with PHP < 7.2. So I guessed it would be better to typehint for now and place this workaround than not typehint those params (or change tests).